uiowa/uiowa
# [CKEditor] media library button style

Resolves https://github.com/uiowa/uiowa/issues/8099. 


# How to test
```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli 
```
Edit a text area and click on the link button.  Confirm that the "Media Library" button is styled with a link color and a hover state.
